docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+`triggers`. The previous tabbable element is the element before the currently focused element in the array stored as
+`tabbableElements` member variable.
+
+### `{inputMapper.base}.forwardTab(that, value, direction)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `direction {String}` The direction for forward tab navigation, i.e. `forwardTab`.
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the next tabbable element using gamepad `buttons` and
+`triggers`. The next tabbable element is the element next to the currently focused element in the array stored as
+`tabbableElements` member variable.
+
+### `{inputMapper.base}.thumbstickTabbing(that, value, speedFactor, invert)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of tab navigation should be increased.
+- `invert {Boolean}` Whether the thumbstick tab navigation should be in opposite order.
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the previous or next tabbable element using gamepad
+`thumbsticks`. It calls the [`{inputMapper.base}.reverseTab`](#inputmapperbasereversetabthat-value-direction) and
+[`{inputMapper.base}.forwardTab`](#inputmapperbaseforwardtabthat-value-direction) invokers according to the direction
+the thumbstick is pressed and works only in **continuous** mode. The direction can be inverted by passing `true` as
+the `invert` argument, and the speed of tab navigation can be changed by changing the value of `speedFactor` argument.
+
+### `{inputMapper.base}.scrollLeft(that, value, speedFactor)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of left scrolling should be increased.
+- Returns: Nothing.
+
+Scrolls the web page in left direction using gamepad `buttons` and `triggers`. The speed of scrolling can be changed by
+modifying the value of `speedFactor` argument.
+
+### `{inputMapper.base}.scrollRight(that, value, speedFactor)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of right scrolling should be increased.
+- Returns: Nothing.
+
+Scrolls the web page in right direction using gamepad `buttons` and `triggers`. The speed of scrolling can be changed by
+modifying the value of `speedFactor` argument.
+
+### `{inputMapper.base}.scrollUp(that, value, speedFactor)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of upward scrolling should be increased.
+- Returns: Nothing.
+
+Scrolls the web page in upward direction using gamepad `buttons` and `triggers`. The speed of scrolling can be changed
```

Comment:
All of these "in X direction" can generally remove "in" and "direction".   Also, fix `buttons` and `triggers` throughout.

Reply:
Much better.  I commented on the two remaining variations that still need to be updated.